src/test/java/Tree/BinaryTreeTest.java

```diff
@@ -0,0 +1,82 @@
+package Tree;
+
+import org.junit.Test;
+import org.junit.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class BinaryTreeTest {
+
+    @Test
+    public void testInOrder() {
+
+        Node root = new Node(4);
+        BinarySearchTree.add(root, 2);
+        BinarySearchTree.add(root, 6);
+        BinarySearchTree.add(root, 1);
+        BinarySearchTree.add(root, 3);
+        BinarySearchTree.add(root, 5);
+        BinarySearchTree.add(root, 7);
+
+        List<Node> expected = Arrays.asList(
+            new Node(1),
+            new Node(2),
+            new Node(3),
+            new Node(4),
+            new Node(5),
+            new Node(6),
+            new Node(7));
+
+        List<Node> actual = BinaryTree.inOrder(root);
+        Assert.assertTrue(actual.size() == 7);
+        Assert.assertEquals(actual, expected);

```

Comment:
You should test on more than one example input; for instance, this should test for more than just a single binary search tree.